docs/pages/database-access/auto-user-provisioning/mysql.mdx

```diff
@@ -111,6 +117,22 @@ Otherwise you may see errors like the following:
 ERROR 1105 (HY000): ERROR 1044 (42000): Access denied for user '<your-teleport-username>'@'%' to database '<database>'
 ```
 
+### Table is read only error
+
+You may encounter the following error when connecting to an AWS RDS Aurora
+reader endpoint:
+```shell
+> tsh db connect mysql --db-name <database> example
```

Comment:
```suggestion
$ tsh db connect --db-name <database> example
```

nit: I think the dollar prompt is more universal.

And the command has two positional args for the database to connect to

Reply:
will update